charts/camunda-platform/README.md

```diff
@@ -475,6 +475,7 @@ For more information about Operate, visit
 | | `image.tag` | Can be set to overwrite the global tag, which should be used in that chart. | `` |
 | | `image.pullSecrets` | Can be set to overwrite the global.image.pullSecrets | `{{ global.image.pullSecrets }}` |
 | | `sidecars` | Can be used to deploy extra containers in the same pod | `[]` |
+| | `extraInitContainers` | Can be used to set up extra init containers for the broker pods, useful for additional exporters | `[ ]` |
```

Comment:
@aabouzaid The reason I used this naming was because I was following the thinking of @jessesimpson36 
He pointed out that it would be a good idea to have both initContainers and extraInitContainers in this [comment](https://github.com/camunda/product-hub/issues/1430#issuecomment-1677900856:~:text=I%20would%20interpret%20extraInitContainers%20to%20be%20a%20list%20appending%20initContainers%20to%20any%20existing%20ones%20inside%20the%20helm%20chart.%20and%20initContainers%20to%20replace%20the%20entire%20list%20of%20initContainers%2C%20removing%20any%20we%20might%20specify%20as%20part%20of%20the%20chart.%20Is%20JPMC%20particular%20about%20which%20one%3F)

Also, `extraInitContainers` already existed in zeebe and zeebe-gateway.
Would you like me to rename all `extraInitContainers` to `initContainers`?
What is your opinion on @jessesimpson36  way of thinking about having both?

Reply:
It totally makes sense to call it `extraInitContainers` but I would keep it `initContainers` as we saw in the past a lot of confusion when users try to set config in Camunda components (Zeebe, Tasklist, etc.) vs. non-Camunda components (Keycloak, Elasticsearch, etc.).

So for 8.3 we will have `initContainers` for all components and add a deprecation notice for `extraInitContainers` in Zeebe and remove it in 8.4 (that means we will use both for Zeebe in 8.3).